### PR TITLE
fix typos in todo project

### LINKED ISSUE #459: errors in todo-project
When describing how to make addTodo function, a couple of confusing typos:

```js
var todos = [{ description: 'make tea' }];
var makeEggs = { description: 'make eggs' };
var updatedTodos = todo.todoFunctions.addTodo(todos, makeEggs);
console.log(updatedTodos)
//  [
//    {
//      description: 'make tea',
//    },
//    {
//      id: 0,
//      description: 'make eggs',
//      done: false,
//     }
//  ]
console.log(todos) // [{ description: 'make eggs' }]
```
The line ```var updatedTodos = todo.todoFunctions.addTodo(todos, makeEggs);``` is weird, I think it should just be ```var updatedTodos = todoFunctions.addTodo(todos, makeEggs);``` because the object in logic.js is just called todoFunctions.

Also (and actually the main one) the line:
```console.log(todos) // [{ description: 'make eggs' }]```
Is presumably to show how the original todo array hasn't changed/been mutated. However the original todo array had ```make tea``` so this is very confusing for students.

Finally, and this may not be a problem, but testing with tape means that the logic.js file needs to be exported, which is catching a few people. It comes up in the Roman Numerals code along a bit, but I don't know if maybe it's worth including:

```js

if (typeof module !== 'undefined') {
  module.exports = todoFunctions;
}
```

at the bottom of the logic.js file, with comments to explain it.

### PR DESCRIPTION
**Pull request by eliascodes**
fixes #459 

Addresses typos identified by @m4v15

### CODE REVIEW

File: coursebook/week-2/project/skeleton/logic.js [hunk 1]
```diff
@@ -38,3 +38,9 @@ var todoFunctions = {
     // hint: array.slice, array.sort
   },
 };
+
+
+// Export todoFunctions for testing with tape
+if (typeof module !== 'undefined') {
```
Comment (m4v15) [inline coursebook/week-2/project/skeleton/logic.js]: Maybe the comment could ask "why do we need this if statement?" or direct them to something like [this](http://www.matteoagosti.com/blog/2013/02/24/writing-javascript-modules-for-both-browser-and-node/)

**Comment by eliascodes** [comment]
@des-des @jsms90 would be grateful of a merge before 2:30pm BST if possible, otherwise before tomorrow morning would also work 🙏

**Comment by eliascodes** [comment]
emergency merge

**Comment by jsms90** [comment]
@eliascodes Sorry, I missed your urgent call on this one.

I trust that you and @des-des know it well enough to know if these changes are appropriate. I'm not remotely familiar with the project, unfortunately.

**Pull request merged**